app/assets/stylesheets/static_pages.scss

```
@@ -0,0 +1,32 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
```

```suggestion
@import 'bootstrap-sprockets';
@import 'bootstrap';
```
code js và css dùng nháy ' thôi em